src/umpire/strategy/DynamicPoolList.cpp

```diff
@@ -113,8 +123,16 @@ void DynamicPoolList::coalesce() noexcept
   dpa.coalesce();
 }
 
-DynamicPoolList::CoalesceHeuristic DynamicPoolList::percent_releasable(
-    int percentage)
+DynamicPoolList::CoalesceHeuristic DynamicPoolList::default_heuristic{percent_releasable(100)};
+
+DynamicPoolList::CoalesceHeuristic DynamicPoolList::releasable_blocks(std::size_t nblocks)
```

Comment:
I think this should be `blocks_releasable` or `num_blocks_releasable` to better match up with `percent_releasable`

Reply:
agreed.  done